schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE vet_clinic.animals (
+    id INT,
```

Comment:
- [ ] [optional] @BHObserver It would be nice if you made your ID the primary key and ensured its not null and is auto incremented